.github/workflows/build_and_test.yml

```diff
@@ -204,3 +204,37 @@ jobs:
           name: ${{ runner.os }}-Unit-Tests
           path: Test*.xml
     if: ${{ always() }}
+
+  package:
+    name: Package
```

Comment:
I don't think `Package` is needed in the "CI" part of the pipeline.
We don't need to generate the package every time someone pushes a new commit to a PR.

And `Package` is also defined (currently double) in the `package_deploy.yml` -- where I think it actually belongs in.

Reply:
Correct. I was originally thinking of having every PR generate a package as well, but this is overkill.

Will cleanup before converting this Draft PR to a real PR.